private_gpt/server/ingest/ingest_service.py

```diff
@@ -6,14 +6,17 @@
 from injector import inject, singleton
 from llama_index import (
     Document,
+    JSONReader,
     ServiceContext,
     StorageContext,
     StringIterableReader,
     VectorStoreIndex,
     load_index_from_storage,
 )
 from llama_index.node_parser import SentenceWindowNodeParser
-from llama_index.readers.file.base import DEFAULT_FILE_READER_CLS
+from llama_index.readers.file.base import (
+    DEFAULT_FILE_READER_CLS as ORIGINAL_DEFAULT_FILE_READER_CLS,
```

Comment:
Small detail to make it more clear for future readers: instead of patching the default file reader we could:
- Keep on importing DEFAULT_FILE_READER_CLS as it was before
- Create our own FILE_READER_CLS var:
FILE_READER_CLS = DEFAULT_FILE_READER_CLS.copy()
FILE_READER_CLS.update(
    {
        ".json": JSONReader,
    }
)

- Use our FILE_READER_CLS in the ingest function body.

Reply:
Done